src/controllers/orderController.js

```diff
@@ -0,0 +1,48 @@
+const { Order, OrderItem } = require('../models/Order');
+const { StatusCodes } = require('http-status-codes');
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+const { BadRequestError } = require('../errors');
+
+const createOrder = async (req, res) => {
+  const { orderItems, subtotal, tax } = req.body;
+
+  try {
+    // Create an array of OrderItem objects
+    const orderItemObjects = orderItems.map(item => new OrderItem(item));
+    // Calculate the total price including tax
+    const total = subtotal + (subtotal * tax);
+
+    // Perform necessary checks and validations
+    if (orderItems.length === 0) {
+      throw new BadRequestError('Unable to create order because the order items are missing.'); 
+    }
+
+    // Create an Order document in the database
+    const order = await Order.create({
+      user: req.user.userId,
+      orderItems: orderItemObjects,
```

Comment:
Does the `Order` model have an `orderItems` field set up in its schema?

Reply:
The code has been fixed in branch PS89, and now the Order model includes the orderItems field as described in the schema.
<img width="506" alt="orderItems" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/9a8847a8-e97e-46bd-b93a-0c34065051b3">